cpp/src/quantiles/quantiles.cu

```diff
@@ -16,196 +16,253 @@
 
 //Quantile (percentile) functionality
 
-#include "quantiles.h"
+#include "quantiles.hpp"

```

Comment:
```suggestion
#include <quantiles.hpp>
```

Reply:
And `<>` on all the rest of the includes here, unless some are local to this file's directory.

Reply:
done